src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -808,11 +809,13 @@ bool MarkDependencyField (TypeDefinition type, string name, in DependencyInfo re
 
 		void LazyMarkCustomAttributes (ICustomAttributeProvider provider)
 		{
-			if (!provider.HasCustomAttributes)
+			if (!_context.CustomAttributes.HasAttributes (provider))
```

Comment:
No need to change this

Reply:
You're right - for now... I reverted the change.